README.md

```diff
@@ -47,25 +47,29 @@ PaperQA depends on some awesome libraries/APIs that make our repo possible. Here
 
 ## Install
 
-You can install PaperQA via pip:
+You can install PaperQA2 via pip:
 
 ```bash
 pip install paper-qa
 ```
 
-PaperQA uses an LLM to operate, so you'll need to either set an appropriate [API key environment variable](https://docs.litellm.ai/docs/providers) (i.e. `export OPENAI_API_KEY=sk-...`) or set up an open source LLM server (i.e. using [llamafile](https://github.com/Mozilla-Ocho/llamafile). Any LiteLLM compatible model can be configured to use with PaperQA.
+PaperQA2 uses an LLM to operate, so you'll need to either set an appropriate [API key environment variable](https://docs.litellm.ai/docs/providers) (i.e. `export OPENAI_API_KEY=sk-...`) or set up an open source LLM server (i.e. using [llamafile](https://github.com/Mozilla-Ocho/llamafile). Any LiteLLM compatible model can be configured to use with PaperQA2.
 
 If you need to index a large set of papers (100+), you will likely want an API key for both [Crossref](https://www.crossref.org/documentation/metadata-plus/metadata-plus-keys/) and [Semantic Scholar](https://www.semanticscholar.org/product/api#api-key), which will allow you to avoid hitting public rate limits using these metadata services. Those can be exported as `CROSSREF_API_KEY` and `SEMANTIC_SCHOLAR_API_KEY` variables.
 
-## What's New?
+## PaperQA2 vs PaperQA
+
+We've been working on hard on engineering it for a while and tried to follow [SemVer](https://semver.org/), meaning we're on version 5. However, we wanted to remark on the fact though that we've exceeded human performance on [many important metrics](https://paper.wikicrow.ai). So we arbitrarily call versions after v5 PaperQA2 and versions before it as PaperQA1 to denote the significant change in performance. We recognize that we are challenged at naming and counting at FutureHouse, so we reserve the right at any time to arbitrarily change the name to PaperCrow.
```

Comment:
> tried to follow [SemVer](https://semver.org/), meaning we're on version 5

Imo following SemVer doesn't mean we are necessarily on version 5. This part could use a reword

Reply:
@jamesbraza - agree and took another pass at this